packages/react-native-codegen/android/gradlew.bat

```diff
@@ -1,89 +1,89 @@
-@rem
-@rem Copyright 2015 the original author or authors.
-@rem
-@rem Licensed under the Apache License, Version 2.0 (the "License");
-@rem you may not use this file except in compliance with the License.
-@rem You may obtain a copy of the License at
-@rem
-@rem      https://www.apache.org/licenses/LICENSE-2.0
-@rem
-@rem Unless required by applicable law or agreed to in writing, software
-@rem distributed under the License is distributed on an "AS IS" BASIS,
-@rem WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-@rem See the License for the specific language governing permissions and
-@rem limitations under the License.
-@rem
-
-@if "%DEBUG%" == "" @echo off
-@rem ##########################################################################
-@rem
-@rem  Gradle startup script for Windows
-@rem
-@rem ##########################################################################
-
-@rem Set local scope for the variables with windows NT shell
-if "%OS%"=="Windows_NT" setlocal
-
-set DIRNAME=%~dp0
-if "%DIRNAME%" == "" set DIRNAME=.
-set APP_BASE_NAME=%~n0
-set APP_HOME=%DIRNAME%
-
-@rem Resolve any "." and ".." in APP_HOME to make it shorter.
-for %%i in ("%APP_HOME%") do set APP_HOME=%%~fi
-
-@rem Add default JVM options here. You can also use JAVA_OPTS and GRADLE_OPTS to pass JVM options to this script.
-set DEFAULT_JVM_OPTS="-Xmx64m" "-Xms64m"
-
-@rem Find java.exe
-if defined JAVA_HOME goto findJavaFromJavaHome
-
-set JAVA_EXE=java.exe
-%JAVA_EXE% -version >NUL 2>&1
-if "%ERRORLEVEL%" == "0" goto execute
-
-echo.
-echo ERROR: JAVA_HOME is not set and no 'java' command could be found in your PATH.
-echo.
-echo Please set the JAVA_HOME variable in your environment to match the
-echo location of your Java installation.
-
-goto fail
-
-:findJavaFromJavaHome
-set JAVA_HOME=%JAVA_HOME:"=%
-set JAVA_EXE=%JAVA_HOME%/bin/java.exe
-
-if exist "%JAVA_EXE%" goto execute
-
-echo.
-echo ERROR: JAVA_HOME is set to an invalid directory: %JAVA_HOME%
-echo.
-echo Please set the JAVA_HOME variable in your environment to match the
-echo location of your Java installation.
-
-goto fail
-
-:execute
-@rem Setup the command line
-
-set CLASSPATH=%APP_HOME%\gradle\wrapper\gradle-wrapper.jar
-
-
-@rem Execute Gradle
-"%JAVA_EXE%" %DEFAULT_JVM_OPTS% %JAVA_OPTS% %GRADLE_OPTS% "-Dorg.gradle.appname=%APP_BASE_NAME%" -classpath "%CLASSPATH%" org.gradle.wrapper.GradleWrapperMain %*
-
-:end
-@rem End local scope for the variables with windows NT shell
-if "%ERRORLEVEL%"=="0" goto mainEnd
-
-:fail
-rem Set variable GRADLE_EXIT_CONSOLE if you need the _script_ return code instead of
-rem the _cmd.exe /c_ return code!
-if  not "" == "%GRADLE_EXIT_CONSOLE%" exit 1
-exit /b 1
-
-:mainEnd
-if "%OS%"=="Windows_NT" endlocal
-
-:omega
+@rem
```

Comment:
Did this file change line endings?

Reply:
Could be. I will figure out what's wrong with it tomorrow, don't want it to be part of the commit.

Reply:
Funny thing tho is, I tried to: a) copy with Github UI manually this file from `master` to this branch b) `git checkout origin master -- <path_to_this_file>`, and both scenarios yield same result - really awkward diff.

Reply:
I've had this same issue with these `*.bat` files. I resolved it for myself with some combination of fiddling with `git config core.autocrlf`, `git rm --cached -r .`, and finally `git reset --hard`, but I don't remember exactly what the silver bullet was. [This StackOverflow](https://stackoverflow.com/questions/1967370/git-replacing-lf-with-crlf) page is what helped me resolve it.

Reply:
The line endings in that file were likely getting affected by the `.gitattributes` files that were present in the repo. It was happening to me as well on another pull I was working on. [A recent commit](https://github.com/facebook/react-native/commit/fb354cba05a71041775a03a3a26122e01b4782a1) fixed this, so hopefully this shouldn't keep happening in the future.

Reply:
@empyrical, are you saying that this change is fine because now line endings are normalized and it won't have this problem in future PRs? Aka, this should have been modified in the change removing gitattributes but wasn't.

Or are you saying that something else is broken here, requires fixing, and this PR shouldn't include these changes?

Reply:
The change to this file should be removed from this diff, IMO.

What I meant to say was Git kept incorrectly trying to change the line endings in this batch file because of those nested `.gitattributes` files, and like how @grabbou was experiencing, no matter how much one tried to undo the line ending changes, it would keep getting changed. This problem shouldn't happen again in the future, because those files were removed.

Reply:
Will update the PR

Reply:
I have cherry-picked a commit that you have referenced @empyrical and it fixed the line endings. Resolving the conversation.

I will rebase this PR on top of master later and the diff should get cleaned up.